src/tlo/methods/consumables.py

```diff
@@ -200,6 +200,7 @@ def _determine_default_return_value(cons_availability, default_return_value):
     def _request_consumables(self,
                              facility_info: 'FacilityInfo',  # noqa: F821
                              item_codes: dict,
+                             optional_item_codes: Optional[dict],
```

Comment:
the type hint says Optional but no default is provided. I think it's good to make it optional though, and this avoids then having to make changes to the test.

```suggestion
                             optional_item_codes: Optional[dict] = None,
```

... and then inside the function

```optional_item_codes = dict() if optional_item_codes is None```